src/en/space-station-14/departments/security/proposals/reduced-metagaming.md

```diff
@@ -0,0 +1,234 @@
+
+### Proposed Solution
+
+Cull the unnecessary fluff items from the set and make them the same level of discoverability. If it's desirable to have the thief hide stuff, include that in the items being recieved; implants require the thief to hide the implanter for example. If the Communicator Set should require the thief to hide a fluff thief figurine for example, instead make the the Master Comms Key come in a box that needs to be hid. 
+
+## The Disguised Items Problem
+
+There are many items in the game that are explicitly designed to look like and sometimes even act like another object. The degree of how effective this is depends on the object. They come in the following categories (some job items not listed):
+
+- Items that can't be found unless directly used
+  - Conducting Gloves
+  - Thieving Gloves
+  - Hypodart
+  - Decoy Nuclear Disk
+  - Explosive Banana
+  - Explosive Wet Floor Sign
+ 
+- Items that can be found if worn
+  - Voice Mask
+  - Chameleon Kit
+  - No-slip Shoes
+
+- Items that can be found if held
+  - Hypopen
+  - Stimkit
+  - Agent ID Card
+  - Lobbying Bundle
+  - Energy Dagger
+  - Cane Sword
+
+- Items that can be found if examined
+  - Extra-Bright Lantern
+  - Blue Boxing Gloves
+  - Dehydrated Space Carp
+
+There is a question to be made what the intended purpose of these disguises are. Something like the Extra-Bright Lantern can only survive a passing glance from crew, a cane sword may be found if an officer is searching thoroughly and a hypodart can't be determined to be a Syndicate object unless explicitly tested. The problem arises when Security is able to be too thorough with a search and find an item they aren't looking for, or when Security knows for certain an item is contraband but can't prove it.
+
+There is also an issue where these disguised items are unknown to crew (until an uplink is found), while the other "obvious" antag items are not. So [crew can have full knowledge](https://forum.spacestation14.com/t/what-is-a-syndicate-item/5363) of emags, webvests, e-swords and northstars, but not about hypopens or thieving gloves. 
+
+### Proposed Solution
+
+This is probably the most comprehensive change and touches on things outside the stated problem above, in the interest of making a more streamlined system and help onboarding new players into contraband gameplay. My proposal is the following:
+
+- Rule change: Crew have knowledge about stealth items.
+- *Non-stealth* items can now be determined to be contraband upon being examined.
+  - Improvised contraband (improvised weapons, shivs, makeshift cuffs etc.) have the following text added: "This item is minor contraband."
+  - Items restricted to departments (insulated gloves, RCDs, scalpels, Security gear etc.) have the following text added: "This item is departmentally restricted."
+  - Command items (HoS' secret orders, Captain's Sabre, Nuke Disk, CE's boots etc.) have the following text added: "This item is restricted to Command."
```

Comment:
I disagree with this once again on opinion (so it's up to you whether or not you agree)
We shouldn't completely eliminate the benefit of having game knowledge by spelling everything out, especially not at a glance like this such that a player would likely ask to examine the item to figure out if it's contraband instead of remembering the name or look or etc. (in other words, this is a variation of the 'minimap problem' of giving the player too much information making them not engage with parts of the game)

If anything, *itemize these checks*, rather literally. Give security a fancy scanning gun or something with a short doafter for example, encourage players to eventually remember these classifications. I'm not against including these kinds of classifications into the game, but I don't think we should shove them into examines.

Reply:
On one hand I agree that this does lower the bar on a certain mechanic, i.e. knowing what is and isn't contraband. On the other hand that's also a mechanic largely enforced via rules exclusively found on an external website outside the game, which isn't the best player experience for a new Sec Cadet. 

This system also acts both ways. It allows non-Sec crew to be aware of what constitutes contraband even if they haven't played Sec or engaged with that system. Which is nice to new players, and also solves the pet peeve of that one tider who refuses to give up their baseball bat because "it's a sports tool, it's not contraband weeeh" so you need to quote the whole contraband list to them.

Reply:
> On one hand I agree that this does lower the bar on a certain mechanic, i.e. knowing what is and isn't contraband. On the other hand that's also a mechanic largely enforced via rules exclusively found on an external website outside the game, which isn't the best player experience for a new Sec Cadet.
> 
> This system also acts both ways. It allows non-Sec crew to be aware of what constitutes contraband even if they haven't played Sec or engaged with that system. Which is nice to new players, and also solves the pet peeve of that one tider who refuses to give up their baseball bat because "it's a sports tool, it's not contraband weeeh" so you need to quote the whole contraband list to them.

Here's an idea: make this a PDA app (NT Certified Object Identification Cloud Service or something else suitably silly), we can offload other "extraneous" information about items into it alongside contraband status in the future if we come up with further uses.

Reply:
making this an app would also tie nicely into existing guidebook systems, and it could potentially pull up related topics for an item.

Reply:
> Here's an idea: make this a **default** PDA app (NT Certified Object Identification Cloud Service or something else suitably silly), we can offload other "extraneous" information about items into it alongside contraband status in the future if we come up with further uses.

Works for me! Afaik the only reason why contraband isn't listed in the guidebook is because... it's not easy to automate? Honestly not sure. Having a comprehensive list in the PDA that looks nice and has a snazzy UI would be pretty neat tbf.

The exact implementation is up in the air but as long as we can mark items in the .ymls with a ContrabandComponent the system can then represent that information in whatever way works best.

Reply:
Automating this is relatively trivial we just don't have a nice system for syncing lists the server invented during startup with the client. (This is something the chemistry guidebook has to have a custom solution for, for example.)

Generalizing that would make it more likely someone would come do it, I might try to do so later.